packages/ssx-sdk/tests/ssx.test.ts

```diff
@@ -42,17 +42,17 @@ test('Instantiate SSX with providers.web3.driver', () => {
 
 // test('Sign-in with Ethereum', () => {
 //   // TODO: sign request with mock provider
-//   // expect(async () => {
-//   //   const config = {
-//   //     providers: {
-//   //       web3: {
-//   //         driver: testingUtils.getProvider(),
-//   //       },
-//   //     },
-//   //   };
-//   //   const ssx = new SSX(config);
-//   //   await ssx.signIn();
-//   // }).not.toThrowError();
+//   expect(async () => {
+//     const config = {
+//       providers: {
+//         web3: {
+//           driver: provider,
+//         },
+//       },
+//     };
+//     const ssx = new SSX(config);
+//     await ssx.signIn();
+//   }).not.toThrowError();
```

Comment:
Did you mean to commit this commented block?

Reply:
This test is commented out, because it fails. It needs the mock provider and can be uncommented. This should be added as part of updating the unit tests